cpp/src/quantiles/tdigest/tdigest_aggregation.cu

```diff
@@ -366,8 +366,8 @@ std::unique_ptr<scalar> to_tdigest_scalar(std::unique_ptr<column>&& tdigest,
  * @param group_cluster_wl    Output.  The set of cluster weight limits for each group.
  * @param group_num_clusters  Output.  The number of output clusters for each input group.
  * @param group_cluster_offsets  Offsets per-group to the start of it's clusters
- * @param has_nulls Whether or not the input contains nulls
- *
+ * @param may_have_empty_clusters Whether or not there could be empty clusters. It should be
+ * set to false only when there is no empty cluster.
```

Comment:
```suggestion
 * @param may_have_empty_clusters Whether or not there could be empty clusters. Must only be
 * set to false when there is no empty cluster, true otherwise.
```

Reply:
Hi @jihoonson, thank you for updating the comment. I have made a small suggestion for better description as per my understanding. Please lmk if this is correct. Good to go otherwise.

Reply:
Thanks for the suggestion. I have updated it as suggested 🙂